Extensions/quick_tags.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Quick Tags **//
-//* VERSION 0.5.2 **//
+//* VERSION 0.5.4 **//
```

Comment:
should change this back to `.3`. Any other PRs that may or may not get pulled will have to be merged or rebased against this one anyway. best to treat them as independent for now, and then resolve conflicts later.